docs/package.json

```
@@ -23,9 +23,7 @@
     "@babel/core": "^7.19.3",
     "@babel/plugin-transform-object-assign": "^7.18.6",
     "@babel/runtime-corejs2": "^7.19.2",
-    "@date-io/core": "^2.15.0",
-    "@date-io/date-fns-jalali": "^2.15.0",
-    "@docsearch/react": "^3.3.0",
+    "@docsearch/react": "^3.2.2",
```

```suggestion
    "@docsearch/react": "^3.3.0",
```

I think this is my mistake. It should not change.